modules/ROOT/pages/dw-core-functions-now.adoc

```diff
@@ -10,14 +10,14 @@ endif::[]
 Returns a `DateTime` object with the current date and time.
```

Comment:
in other places where we have date-time, date and time is an easier-to-read choice?